src/components/FilmBlock.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+export class FilmBlock extends React.PureComponent {
+    render() {
+        return(
+            <React.Fragment>
```

Comment:
React fragment is used for avoiding some weird design in components. Why you need it here ? You can return just <div>...</div>